<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QMainWindow" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BIAS Camera Window</string>
  </property>
  <widget class="QWidget" name="centralwidgetPtr_">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="imageLabelPtr_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="connectButtonPtr_">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButtonPtr_">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="cameraInfoLabelPtr_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Camera Info</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>599</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubarPtr_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Logging</string>
    </property>
    <widget class="QMenu" name="menuFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <addaction name="actionAVI"/>
     <addaction name="actionFMF"/>
     <addaction name="actionUFMF"/>
    </widget>
    <widget class="QMenu" name="menuTimer">
     <property name="title">
      <string>Timer</string>
     </property>
     <addaction name="actionEnabled"/>
     <addaction name="actionTimerSettings"/>
    </widget>
    <addaction name="actionVideo_File"/>
    <addaction name="menuFormat"/>
    <addaction name="menuTimer"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Video Mode</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProperties">
    <property name="title">
     <string>Framerate</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuProperties_2">
    <property name="title">
     <string>Properties</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuProperties"/>
   <addaction name="menuProperties_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbarPtr_"/>
  <action name="actionLogging">
   <property name="text">
    <string>Logging</string>
   </property>
  </action>
  <action name="actionVideo_File">
   <property name="text">
    <string>Video File...</string>
   </property>
  </action>
  <action name="actionAVI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>avi</string>
   </property>
  </action>
  <action name="actionFMF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>fmf</string>
   </property>
  </action>
  <action name="actionUFMF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ufmf</string>
   </property>
  </action>
  <action name="actionTimer">
   <property name="text">
    <string>Timer</string>
   </property>
  </action>
  <action name="actionEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </action>
  <action name="actionTimerSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
